pkg/3scale/amp/component/system.go

```diff
@@ -927,9 +1059,10 @@ func (system *System) SidekiqDeployment(containerImage string) *k8sappsv1.Deploy
 							Command: []string{
 								"bash",
 								"-c",
-								"bundle exec sh -c \"until rake boot:redis && curl --output /dev/null --silent --fail --head http://system-master:3000/status; do sleep $SLEEP_SECONDS; done\"",
+								"bundle exec sh -c \"until rake boot:redis && curl --insecure --output /dev/null --silent --fail --head http://system-master:3000/status; do sleep $SLEEP_SECONDS; done\"",
```

Comment:
why are we bypassing ssl cert validation here?

Reply:
@MStokluska, "--insecure" means that curl will not verify the SSL certificate of the server it's connecting to. This is typically used when the server has a self-signed certificate or the certificate is not trusted by the local machine.  
I will try to retest if it will work for me without "--insecure" but not sure.
Thank you for comment

Reply:
I had to do this on my PR as well even though I didn't have redis tls enabled .